full_bubble_model.py

```diff
@@ -996,45 +1044,46 @@ def plot(cpar, t_int=np.array([0.0, 1.0]), n=5.0, base_name='', format='png', LS
 # Solve
     if type(cpar) == dict:
         cpar = dotdict(cpar)
-
-    num_sol, error_code, elapsed_time = solve(cpar, t_int, LSODA_timeout, Radau_timeout, extra_dims, print_errors=True)
+    compression = 1 if base_name == '' else 0
```

Comment:
`de.plot()` will use compression=1 when you are just plotting, and compression=0 when you save the plot (alias when base_name is provided as an argument)

compression=1 will improve plotting speed by using ~1/10th the steps, but it slightly noticeable (top of the waves, where R is maximal might not be fully smooth) - the difference might be seconds when a lot of steps are used